### PR TITLE
G6 Develop into Develop

### PR DESCRIPTION
**Pull request by Thomlam**

### CODE REVIEW

File: front-end/src/components/Agility.vue [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+		view: pixi.value,
+		resolution: window.devicePixelRatio || 1, // Explain to me what that shit ? (It's troll, logic to convert Pixel 1/1)
+		//Set transparent background
+		autoDensity: true, // Gné ?
+		width: window.innerWidth,
+		height: window.innerHeight,
+
+	})
+	mainCanvas = app;
+
```
Comment (Loule95450) [inline front-end/src/components/Agility.vue:131]: C'est quoi ça ?

**Review by Maghwyn — APPROVED** [review]
To be linted later on.

File: front-end/src/main.ts [hunk 2]
```diff
@@ -16,6 +19,10 @@ function bootVueApp() {
 	app.use(router);
 	app.use(store);
 	app.mount('#app');
+
+
```
Comment (Loule95450) [inline front-end/src/main.ts:23]: 22, 23 cleanup plz

File: front-end/src/router/index.ts [hunk 3]
```diff
@@ -7,6 +7,8 @@ const routes: Array<RouteRecordRaw> = [
 		children: [
 			{ path: '', component: () => import('../views/HomeView.vue') },
 			{ path: '/about', component: () => import('../views/AboutView.vue') },
+			{ path: '/agility', component: () => import('../views/AgilityView.vue') },
+
```
Comment (Loule95450) [inline front-end/src/router/index.ts:11]: 11 cleanup plz

File: front-end/src/main.ts [hunk 4]
```diff
@@ -3,10 +3,13 @@ import App from './App.vue';
 import router from './router';
 import store from './store';
 
+
 import './styles/tailwindcss.css';
 import './styles/style.css';
 import './styles/layout.css';
 
+
+
```
Comment (Loule95450) [inline front-end/src/main.ts:12]: 11, 12 cleanup plz

**Review by Loule95450 — CHANGES_REQUESTED** [review]
Cleanup plz

**Review by Loule95450 — APPROVED** [review]
👍

**Pull request merged**